scripts/zones/Northern_San_dOria/npcs/Ailbeche.lua

```diff
@@ -11,26 +11,39 @@ require("scripts/globals/settings");
 require("scripts/globals/keyitems");
 require("scripts/globals/titles");
 require("scripts/globals/quests");
+
 require("scripts/globals/shop");
 -----------------------------------
 
-function onTrade(player,npc,trade)
-    -- "Flyers for Regine" conditional script
-    local FlyerForRegine = player:getQuestStatus(SANDORIA,FLYERS_FOR_REGINE);
+-- Helper constants in order to keep it straight about what part
+-- of the quest we are working on
+BOYS_DREAM_WILL_BEGIN_ACCEPTING_BUG = 3
```

Comment:
1. Global vars are bad.
2. Please use comments to explain the steps when needed, rather than turning the sql variable value into multiple lua variables..

Reply:
Is there a reason? The variables are more self describing. The fact that they're globals sucks but otherwise, it's definitely easier and less redundant, no?

That being said, there is consistency to value.